dataset-snippets/summarize_a_dataframe.py

```diff
@@ -0,0 +1,7 @@
+# Use snippet 'summarize_a_dataframe' to display summary statistics for a dataframe.
+# It assumes snippet 'Setup' has been executed.
+# See also https://towardsdatascience.com/exploring-your-data-with-just-1-line-of-python-4b35ce21a82d
+
+
+## -----[ CHANGE THE DATAFRAME NAME(S) TO MATCH YOURS FROM DATASET BUILDER] -----
+YOUR_DATASET_NAME_person_df.profile_report()
```

Comment:
FWIW, this just hung for me. May want to comment it is very slow (though I've never actually seen it succeed yet)

Reply:
Agreed, not a good default. I updated it to default max of 10,000 rows. Added a comment about the time for larger dataframes.